src/drive/mobile/lib/cozy-helper.js

```diff
@@ -49,7 +49,10 @@ export const initBar = () => {
     appEditor: 'Cozy',
     iconPath: require('../../../../targets/drive/vendor/assets/app-icon.svg'),
     lang: getLang(),
-    replaceTitleOnMobile: true
+    cozyURL: client.getUrl(),
```

Comment:
So why did I provide the URL and the token to cozy-bar in this PR?

On mobile, every view has something in the `BarRight`, except the Settings views, where there's nothing. When there's nothing to display, the fallback is the [Nav](https://github.com/cozy/cozy-bar/blob/master/src/components/Bar.jsx#L106), but trying to render that component without the list of apps breaks everything. So with the cozy url and the token, the bar can fetch the apps and not disply anything at all in the `BarRight` section when needed.

Why didn't we care until now? Because [`displayMobile`](https://github.com/cozy/cozy-bar/blob/master/src/components/Bar.jsx#L105) was set to false.

We have another upcoming card on this subject this sprint to tidy things up.